code/modules/clothing/shoes/miscellaneous.dm

```diff
@@ -343,3 +343,11 @@
 		recharging_time = world.time + recharging_rate
 	else
 		to_chat(user, "<span class='warning'>Something prevents you from dashing forward!</span>")
+
+/obj/item/clothing/shoes/ducky
+	name = "rubber ducky shoes"
+	desc = "Two rubber duckies made to fit."
+	icon_state = "ducky"
+	item_state = "ducky"
+	var/footstep = 1
```

Comment:
What is this var for? It's not used anywhere.

Reply:
well I know next to nothing about byond code, I just noticed that for the other shoes with sounds, like jackboots and clownshoes, had it. So I copied it over. I never tested to see if it still made the noises without it, I just assumed it was necessary.

Reply:
OK so sounds work fine without that var, and it is seemingly totally pointless. (for rubber ducky shoes at least)

Reply:
Aha I see. After some research, those vars on the other shoes are also unused, because shoe step sound is handled elsewhere and it never even uses the `footstep` var.

Reply:
must be an artifact or something